recipe/meta.yaml

```diff
@@ -1,19 +1,18 @@
-{% set version = "5.4.1" %}
+{% set version = "6.0" %}
 
 package:
   name: pyyaml
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/P/PyYAML/PyYAML-{{ version }}.tar.gz
-  sha256: 607774cbba28732bfa802b54baa7484215f530991055bb562efbed5b2f20a45e
+  sha256: 68fb519c14306fec9720a2a5b45bc9f0c8d1b9c72adf45c37baedfcd949c35a2
   patches:
     - 0001-Ensure-we-do-not-end-up-wih-CRLF-line-endings-on-tes.patch
-    # Can be removed if https://github.com/yaml/pyyaml/pull/542 is merged
-    - 0002-Close-resources.patch
```

Comment:
@mattip looks like we still need this patch. How are things upstream? The patch is quite massive to keep updating it every new release. It would be nice to get it merged :smile:

Reply:
This PR deletes it and still fails :-(

Reply:
Oh. I did not check the logs a second time. My bad! Thanks!!
Let me fix that skipping `pip check`.